docs/design/communication.md

```
@@ -120,32 +111,29 @@ sole transport for intra-organization messages.
-|                             v     v                                   |
-|                       +--------------------+                          |
-|                       |   Message Bus      |                          |
-|                       |  (internal,        |                          |
-|                       |   unchanged)       |                          |
-|                       +--+----+----+----+--+                          |
-|                          |    |    |    |                             |
-|                       +--v-+--v-+--v-+--v-+                           |
-|                       | A1 | A2 | A3 | A4 |  Internal Agents          |
-|                       +----+----+----+----+                           |
-+-----------------------+-----------------------------------------------+
+```d2
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)